app/model/IdUserOps.scala

```diff
@@ -8,12 +8,12 @@ object IdUserOps {
 
     def billingAddress: Address = {
       Address(
-        lineOne = u.privateFields.billingAddress1.getOrElse(""),
-        lineTwo = u.privateFields.billingAddress2.getOrElse(""),
-        town = u.privateFields.billingAddress3.getOrElse(""),
-        countyOrState = u.privateFields.billingAddress4.getOrElse(""),
-        postCode = u.privateFields.billingPostcode.getOrElse(""),
-        countryName = u.privateFields.billingCountry.getOrElse("")
+        lineOne = "",
+        lineTwo = "",
+        town = "",
+        countyOrState = "",
+        postCode = "",
+        countryName = ""
```

Comment:
AFAIR billing address is no longer stored by identity, so it would have always been "" without the model changes.  This is used to prefill the checkout for logged in users (of which there are none anyway as this page is for CSRs only.)